packages/runtime/src/core.ts

```diff
@@ -92,6 +93,30 @@ export class FederationHost {
       ],
       void
     >('loadRemote'),
+    interfacePrefetch: new AsyncHook<
```

Comment:
i think the hook name is not easy to understand , have any ideas to name it ? @ScriptedAlchemy @zhoushaw 

prefetch: help users prefetch interfaces/requests/apis , and speed up the page/module load , make it look like non-request

preload: preload module assets , which can help to reduce waiting time

Reply:
this name maybe is clear enough?